rpc/websocket_test.go

```diff
@@ -246,7 +247,9 @@ func wsPingTestHandler(t *testing.T, conn *websocket.Conn, shutdown, sendPing <-
 				t.Errorf("got pong with wrong data %q", data)
 			}
 			wantPong = ""
-			sendResponse = time.NewTimer(200 * time.Millisecond).C
+			timer := time.NewTimer(200 * time.Millisecond)
+			defer timer.Stop()
```

Comment:
This is not a good idea because the deferred calls will be collected in the loop.
Since we only need one timer here, the logic needs to change to stop the timer once
when the loop exits.

Reply:
I see , maybe put timer out of loops will be ok